site/_docs/reference.md

```diff
@@ -1751,6 +1751,8 @@ and `LISTAGG`).
 | SUM( [ ALL &#124; DISTINCT ] numeric)         | Returns the sum of *numeric* across all input values
 | MAX( [ ALL &#124; DISTINCT ] value)           | Returns the maximum value of *value* across all input values
 | MIN( [ ALL &#124; DISTINCT ] value)           | Returns the minimum value of *value* across all input values
+| BOOL_AND(value)                               | Returns true if all *values* are true
+| BOOL_OR(value)                                | Returns true if at least one value of *value* is true
```

Comment:
Does BOOL_AND/OR support `[DISTINCT]` keyword?

Reply:
Yes, it should be ignored.

Reply:
Here is my mistake. I checked [Redshift Document](https://docs.aws.amazon.com/redshift/latest/dg/r_BOOL_OR.html), `[DISTINCT | ALL] ` should be supported.